assets/evaluation_on_cloud/environments/evaluations-built-in/context/requirements.txt

```diff
@@ -1,4 +1,4 @@
-azure-ai-evaluation==1.0.0b5
```

Comment:
as discussed offline, lets remove it from this PR!

Reply:
Removed